src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -80,15 +80,19 @@ object Writer {
     case class CircularExports(
```

Comment:
is this the correct directory to put this script in? seems odd to check in something to the path github/{your_username}/...